tests/exploreServices.spec.ts

```diff
@@ -15,10 +15,12 @@ test.describe('explore services page', () => {
   test('should filter service labels on search', async ({ page }) => {
     await explorePage.servicesSearch.click();
     await explorePage.servicesSearch.pressSequentially('mimir');
+
+    await page.getByTestId('data-testid Panel header mimir-ingester').first().scrollIntoViewIfNeeded()
     // service name should be in time series panel
-    await expect(page.getByTestId('data-testid Panel header mimir-ingester').nth(0)).toBeVisible();
+    expect(await page.getByTestId('data-testid Panel header mimir-ingester').nth(0)).toBeDefined()
     // service name should also be in logs panel, just not visible to the user
-    await expect(page.getByTestId('data-testid Panel header mimir-ingester').nth(1)).toBeVisible();
+    expect(await page.getByTestId('data-testid Panel header mimir-ingester').nth(1)).toBeDefined();
```

Comment:
I guess defined is less flaky than visible?

Reply:
I saw some executions in ci where the volume was different then local and it wasn't rendered until scrolling to it, but it should be in the viewport now... I'll double check that this is needed

Reply:
I changed it back to is visible, passed 10x locally, let's see aboot ci

Reply:
I changed it back, seems to be passing? Could be that I changed it before adding the scrollTo and never changed it back. I'll re-run in CI a few times and if it flakes we can take another look